buildenv/jenkins/JenkinsfileBase

```diff
@@ -832,6 +833,17 @@ def getJenkinsDomain() {
 	return domainName
 }
 
+def archiveTestenvFile() {
+	def testenvFile = "aqa-tests/testenv/testenv.properties";
+	if (!params.ARTIFACTORY_SERVER) {
+		echo "ARTIFACTORY_SERVER is not set. Saving testenv.properties file on jenkins."
+		archiveArtifacts artifacts: testenvFile, fingerprint: true, allowEmptyArchive: true
+	} else {
+		def pattern = "${env.WORKSPACE}/${testenvFile}"
+		uploadToArtifactory(pattern)
+	}
+}
+
```

Comment:
Please remove the changes for archiving testenv.properties as the change is merged.